System.IO.Ports/SerialDeviceEventListener.cs

```diff
@@ -9,7 +9,7 @@ namespace System.IO.Ports
     internal class SerialDeviceEventListener : IEventProcessor, IEventListener
     {
         // Map of serial device numbers to SerialDevice objects.
-        private readonly ArrayList _serialDevicesMap = new();
+        private readonly ArrayList _serialDevicesMap = new ArrayList();
```

Comment:
note: the linter does not support yet this new way but if you do `new ();` it will work but ctrl + k + d will make it again with the parentheses attached to the new.

Reply:
Reverted.